src/test/java/co/cask/hydrator/python/transform/PythonTransformInterpretedTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.python.transform;
+
+public class PythonTransformInterpretedTest extends PythonTransformNativeTest {
```

Comment:
It's kind of odd for Intepreted to extend Native. I think it makes more sense to have an abstract base class where getExecutionMode() is abstract, and have an Interpreted one and a Native one that both extend the base one. The base class should have tests that work in both execution modes, and Native can add the ones that are only applicable to native mode and Interpreted add ones that are only applicable to interpreted mode.